core/core.schema.go

```diff
@@ -37,12 +40,24 @@ extend type Query {
 "Core API"
 type Core {
 	"Fetch an OCI image"
-	image(ref: String!): Filesystem!
+	image(ref: String!): Image!
 
 	"Fetch a git repository"
 	git(remote: String!, ref: String): Filesystem!
 }
 
+"Image filesystem and initial exec inputs, e.g. an OCI image"
```

Comment:
> Side note: this big old string seems like it could be replaced with a go:embed so we can put the schema in a separate file with syntax highlighting/etc. I'd be interested in doing that as a separate PR. Any reason not to?

👍 definitely